src/Transports/MassTransit.ActiveMqTransport/Topology/Topologies/ActiveMqConsumeTopology.cs

```diff
@@ -71,17 +74,20 @@ public void Bind(string topicName, Action<ITopicBindingConfigurator> configure =
         public override string CreateTemporaryQueueName(string tag)
         {
             var result = base.CreateTemporaryQueueName(tag);
-            return new string(result.Where(c => c != '.').ToArray());
+            var tempName = new string(result.Where(c => c != '.').ToArray());
+            return tempName;

```

Comment:
Leftover for some reason.

Reply:
new commits have different implementation